src/test/java/com/google/api/codegen/testdata/library_pkg.yaml

```diff
@@ -0,0 +1,48 @@
+# Common configuration
+short_name: library
+major_version: v1
+proto_path: google/library
+author: Google, Inc.
+email: googleapis-packages@google.com
+homepage: https://github.com/googleapis/googleapis
+license: Apache-2.0
+
+# Language-dependent configuration
+package_name:
+  default: google-cloud-library-v1
```

Comment:
Respectfully disagree: there's coverage for language-specific configuration elsewhere in this file. This tests the logic for handling the "default" case.